src/datagrid/DataGrid.tsx

```diff
@@ -1034,43 +1199,62 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
                     ClassNames.DATAGRID_ROW,
                     ClassNames.DATAGRID_NG_STAR_INSERTED,
                     !disableRowSelection && isSelected && ClassNames.DATAGRID_SELECTED,
+                    isRowWithDetailPane &&
+                        detailPaneData &&
+                        detailPaneData.isOpen &&
+                        ClassNames.DATAGRID_ROW_DETAIL_OPEN,
                     className,
                 ])}
                 aria-owns={"clr-dg-row" + index}
                 style={rowStyle}
                 key={"row-" + index}
             >
-                <div
-                    className={classNames([
-                        `ng-tns-c96-${index}`,
-                        isExpandableRow && ClassNames.DATAGRID_EXPAND_ANIMATION,
-                    ])}
-                >
+                {isExpandableRow ? (
                     <div
-                        className={classNames([ClassNames.DATAGRID_ROW_MASTER, ClassNames.DATAGRID_NG_STAR_INSERTED])}
-                        role="row"
+                        className={classNames([
+                            `ng-tns-c96-${index}`,
```

Comment:
how this c96 calculated ? does it have any significance ?

Reply:
This is CSS class name coming from clarity. c96 is same for all rows not calculated